skype/skype-ps/skype/Get-CsCallerIdPolicy.md

```diff
@@ -8,7 +8,7 @@ schema: 2.0.0
 # Get-CsCallerIdPolicy
 
 ## SYNOPSIS
-Provide the topic introduction here.
+This cmdlet has been deprecated please use Get-CsCallingLineIdentity to create new caller id policy.
```

Comment:
Should not say "to create new".. should say "to get"

Reply:
I see this same comment repeated for the other CsCallerId cmdlets too.   Let's use a more generic comment:

"The CsCallerIdPolicy cmdlets have been deprecated.  Please use the CsCallingLineIdentity cmdlets to manage caller ID policies for your organization.

Reply:
done, thanks!